### ISSUE TITLE
Unknown error

### ISSUE THREAD

**Issue by theranbrig** [deanishe/alfred-repos#22]
I keep getting errors with the following output:

```
[22:48:35.460] Git Repos[Keyword] Processing complete
[22:48:35.479] Git Repos[Keyword] Passing output '' to Run Script
[22:48:35.742] ERROR: Git Repos[Run Script] .
22:48:35 workflow.py:2061 DEBUG    ---------- Git Repos (3.0.0) ----------
22:48:35 workflow.py:1468 DEBUG    reading settings from /Users/theranbrigowatz/Library/Application Support/Alfred/Workflow Data/net.deanishe.alfred-git-repos/settings.json
22:48:35 workflow.py:2343 DEBUG    update check not due
22:48:35 workflow.py:2225 DEBUG    last run version: None
22:48:35 workflow.py:2080 ERROR    not a Version instance: None
Traceback (most recent call last):
  File "/Users/theranbrigowatz/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.096FD77C-8226-4021-9611-C5B4BB4445E4/workflow/workflow.py", line 2073, in run
    func(self)
  File "repos.py", line 369, in main
    if wf.last_version_run < Version('2'):
  File "/Users/theranbrigowatz/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.096FD77C-8226-4021-9611-C5B4BB4445E4/workflow/update.py", line 322, in __gt__
    raise ValueError('not a Version instance: {0!r}'.format(other))
ValueError: not a Version instance: None
22:48:35 workflow.py:2082 INFO     for assistance, see: https://github.com/deanishe/alfred-repos/issues
22:48:35 workflow.py:2103 DEBUG    ---------- finished in 0.006s ----------
```

My settings file is very basic:
```
{
  "app_cmd": "Terminal", 
  "app_default": "Finder", 
  "global_exclude_patterns": [], 
  "search_dirs": [
    {
      "depth": 2, 
      "excludes": [
        "tmp", 
        "bad/smell/*"
      ], 
      "name_for_parent": 1, 
      "path": "~/WebDev/projects"
    }
  ]
}
```

**Comment by deanishe** [comment]
Try running `repos workflow:delcache` to clear the cache.

**Comment by theranbrig** [comment]
<img width="563" alt="Screen Shot 2019-11-06 at 11 21 18 PM" src="https://user-images.githubusercontent.com/29407645/68306109-3fbf3980-00ec-11ea-9e53-df4a2ba260a3.png">

I just get this.

**Comment by deanishe** [comment]
Please enter the command I told you to.

**Comment by theranbrig** [comment]
<img width="568" alt="Screenshot_2019-11-07 08 33 43_inKrQg" src="https://user-images.githubusercontent.com/29407645/68347158-69eb1880-0139-11ea-94f9-932497b04598.png">
Sorry.  Posted the wrong screenshot.  Still have the same error.  Unless I am entering it in in the wrong place or something?

**Comment by deanishe** [comment]
Should be fixed in [v3.0.1](https://github.com/deanishe/alfred-repos/releases/tag/v3.0.1).

In future, please don't post screenshots: they're pretty much useless. Always post the actual error from Alfred's debugger or the log file.

**Comment by theranbrig** [comment]
Normally I wouldn't.  But I wasn't even getting anything in the debugger.  Thanks for the changes.  Cheers.
